R/ml_feature_transformation.R

```diff
@@ -391,3 +391,30 @@ ft_regex_tokenizer <- function(x,
 #     setBinary      = ensure_scalar_boolean(binary)
 #   ))
 # }
+
+#' Feature Tranformation -- CountVectorizer
+#'
+#' Extracts a vocabulary from document collections.
+#'
+#' @template roxlate-ml-transformation
+#'
+#' @export
+ft_count_vectorizer <- function(x,
+                                input.col = NULL,
+                                output.col = NULL,
+                                min_df = NULL,
```

Comment:
Can we use `min.df` rather than `min_df` here? I've adopted a convention in the various ML functions that parameter names use `.` as a separator rather than `_`.